docs/config.md

```diff
@@ -109,6 +109,8 @@ The following settings are available:
   :::
 : When enabled, OCI (and Docker) container images are pulled and converted to the SIF format by the Apptainer run command, instead of Nextflow (default: `false`).
 
+  Note: leave `ociAutoPull` disabled if willing to build a Singularity/Apptainer native image with Wave (see the {ref}`wave-singularity` page).
```

Comment:
Let's turn these into proper notes with `:::{note} ...`. Also the link is to a "section" rather than a "page"

Reply:
done!